src/widgets/mentors-benefit/ui/mentors-benefit.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames/bind';
+import { WidgetTitle } from '@/shared/ui/widget-title';
+import { mentorsBenefitData } from 'data';
+
+import styles from './mentors-benefit.module.scss';
+
+const cx = classNames.bind(styles);
+
+type MentorsBenefitsProps = {
+  benefits: string[];
+  lang?: 'en' | 'ru';
+};
+
+const countSymbols = 60;
+
+export const MentorsBenefits = ({ benefits, lang = 'en' }: MentorsBenefitsProps) => {
+  return (
+    <section className={cx('container')}>
+      <div className={cx('about-mentorship', 'content')}>
+        <WidgetTitle className={cx('title')}>{mentorsBenefitData[lang].header}</WidgetTitle>
+        <ul className={cx('benefits')}>
+          {benefits.map((info, index) => {
+            let classNameWidth = ((info.length > countSymbols) ? 'benefit-long' : 'benefit-short');
+
+            return (
+              <li key={index} className={cx('benefit', classNameWidth)}>
```

Comment:
```suggestion
            return (
              <li
                key={index}
                className={cx('benefit', 'benefit-short', { 'benefit-long': info.length > shortBenefitMaxChars })}
              >
```

Reply:
This is how the style override will happen, I didn't mean for it to happen.
![image](https://github.com/user-attachments/assets/c2a2b0eb-bf60-40bc-bc11-4ac9d8e21d09)